### PR TITLE
tutorial script

### PR DESCRIPTION
**Pull request by Lukong123**
Still a newbie, please this script is open to corrections.

### CODE REVIEW

File: 2022-round-1/Lukong123/nx_tutorial_script.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+# Calculating shortest path between all pairs of node
+
+path = dict(nx.all_pairs_shortest_path(DG))
+print(path) 
+
+# plotting graph using networkx.draw
+
+my_graph = nx.complete_graph(DG)
+nx.draw(my_graph)
```
Comment (MridulS) [inline 2022-round-1/Lukong123/nx_tutorial_script.py:36]: This looks great!

There isn't a need to make a complete graph to use `nx.draw`.
Any particular reason why is the directed graph first cast into a `complete_graph`?

Something like `nx.draw(DG)` should be enough here :)

Comment (Lukong123) [reply 2022-round-1/Lukong123/nx_tutorial_script.py:36]: Okay sir, copied that.

Comment (Lukong123) [reply 2022-round-1/Lukong123/nx_tutorial_script.py:36]: No sir, there is no particular reason why I cast it to a `complete_graph`

**Comment by Lukong123** [comment]
@MridulS I have rectified the code.

**Pull request merged**